packages/runtime/src/helpers/edge.ts

```diff
@@ -476,6 +489,7 @@ export const writeEdgeFunctions = async ({
       function: 'ipx',
       name: 'next/image handler',
       path: nextConfig.images.path || '/_next/image',
+      generator: `${nextjsPluginVersion}`,
```

Comment:
This doesn't need to be a template string now and you can just use the variable.

```suggestion
      generator: nextjsPluginVersion`,
```